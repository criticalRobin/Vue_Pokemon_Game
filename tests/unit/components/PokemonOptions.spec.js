import { shallowMount } from "@vue/test-utils";
import PokemonOptions from "@/components/PokemonOptions.vue";
import { pokemonsArr } from "../mocks/pokemons.mock";

describe("pruebas unitarias componente PokemonOptions", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(PokemonOptions, {
      props: {
        pokemonsArr,
      },
    });
  });

  test("debe hacer match con el snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
