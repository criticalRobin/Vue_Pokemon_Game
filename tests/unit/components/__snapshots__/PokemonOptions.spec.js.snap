// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas unitarias componente PokemonOptions debe hacer match con el snapshot 1`] = `
<div class="options-container">
  <ul>
    <li>bulbasaur</li>
    <li>ivysaur</li>
    <li>venusaur</li>
    <li>charmander</li>
  </ul>
</div>
`;
